docs/content/preview/yugabyte-cloud/cloud-monitor/cloud-queries-slow.md

```diff
@@ -12,30 +12,48 @@ menu:
 type: docs
 ---
 
-Evaluate the performance of slow queries that have run on your cluster using the **Slow Queries** option on the cluster **Performance** tab. Use this data to:
+Evaluate the performance of slow queries that have run on your cluster using the **Slow Queries** option on the cluster **Performance** tab.
+
+{{< youtube id="hXphqRCQImQ" title="Monitor and optimize queries in YugabyteDB Aeon" >}}
+
+Use this data to:
 
 - Visually identify slower running database operations.
 - Evaluate query execution times over time.
 - Discover potential queries for [tuning](../../../explore/query-1-performance/).
-
-{{< youtube id="hXphqRCQImQ" title="Monitor and optimize queries in YugabyteDB Aeon" >}}
+- View latency percentile metrics, including P99, P95, P90, P50, and P25 percentiles.
 
 Slow queries are not available for YCQL.
 
-![Cluster Slow Queries](/images/yb-cloud/managed-monitor-slow-queries.png)
+## View Slow Queries
 
-To filter the query list, enter query text in the filter field. To sort the list by column, click the column heading. Click **Edit Options** to select the columns to display.
+To view slow queries, navigate to the cluster **Performance** tab and choose **Slow Queries**.
 
-To reset the slow queries list, click **Reset**.
+![Cluster Slow Queries](/images/yb-cloud/managed-monitor-slow-queries.png)
 
-To view query details, click the right-arrow button for the query in the list to display the **Query Details** sheet.
+To filter the query list, enter query text in the filter field. To sort the list by column, click the column heading.
 
-The following table describes the **Slow Queries** column values.
+Click **Edit Options** to select the columns to display. The following table describes the **Slow Queries** column values.
 
 | Column          | Description                                                  |
 | --------------- | ------------------------------------------------------------ |
+| Query           | The query statement.               |
 | User            | The name of role used to access YSQL database.               |
-| Total time      | The total duration (in milliseconds) of all iterations this query has taken. |
+| Database        | The YSQL database used by the query.                         |
+| Avg time        | The average execution time (in milliseconds) for this query. |
 | Count           | The total number of times this type of query has executed.   |
+| Total time      | The total duration (in milliseconds) of all iterations this query has taken. |
 | Rows            | The total number of database table rows returned across all iterations of this query |
```

Comment:
The total number of database table rows returned across all iterations of this query.